### PR TITLE
dosbox-staging 0.81.0 (new cask)

### PR DESCRIPTION
**Pull request by Burrito78**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by krehel** [comment]
@Burrito78 please delete the extra file and run `brew style --fix` on the Cask and push the updates. Thanks.

**Comment by Burrito78** [comment]
> @Burrito78 please delete the extra file and run `brew style --fix` on the Cask and push the updates. Thanks.

Is it OK now? I did delete the extra file through GitHub.com.

**Comment by SMillerDev** [comment]
> Is it OK now? I did delete the extra file through GitHub.com.

Please also run `brew style --fix`.

File: Casks/d/dosbox-staging.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask "dosbox-staging" do
+  version "0.80.1"
```
Comment (kklobe) [inline Casks/d/dosbox-staging.rb:2]: should be 0.81.0

**Comment by bevanjkay** [comment]
Is it necessary to have this included in `homebrew-core` and `homebrew-cask`?
How do the two packages differ?

https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/d/dosbox-staging.rb

**Comment by Burrito78** [comment]
Having it installed as an .app bundle like any other macOS GUI application is the typical use case of DOSBox Staging.

I'd say that 95% of users (myself included) would want it that way instead of it being a terminal application.
I guess being used from the terminal is maybe preferred by Linux users but that is not what Homebrew is about.

**Comment by SMillerDev** [comment]
> I guess being used from the terminal is maybe preferred by Linux users but that is not what Homebrew is about.

It's definitely also what Homebrew is about.

In https://github.com/Homebrew/homebrew-core/pull/57450 it seems like this is not specifically for Linux.

**Comment by Burrito78** [comment]
> > I guess being used from the terminal is maybe preferred by Linux users but that is not what Homebrew is about.
> 
> It's definitely also what Homebrew is about.
> 
> In [Homebrew/homebrew-core#57450](https://github.com/Homebrew/homebrew-core/pull/57450) it seems like this is not specifically for Linux.

Yes, sorry. I didn't make myself clear. What i meant was "Homebrew isn't about Linux" and not "Homebrew isn't about command line apps".

I would absolutely prefer to keep both versions in Homebrew to give users a choice here.

**Comment by SMillerDev** [comment]
> Yes, sorry. I didn't make myself clear. What i meant was "Homebrew isn't about Linux" and not "Homebrew isn't about command line apps".

I understood that it was about Linux, and I still strongly disagree and so do our analytics: https://formulae.brew.sh/analytics/os-version/365d/

**Comment by Burrito78** [comment]
> > Yes, sorry. I didn't make myself clear. What i meant was "Homebrew isn't about Linux" and not "Homebrew isn't about command line apps".
> 
> I understood that it was about Linux, and I still strongly disagree and so do our analytics: https://formulae.brew.sh/analytics/os-version/365d/

Yes, you are correct. I forgot that it's also for Linux. Sorry.

**Comment by bevanjkay** [comment]
Thanks for the explanation. I am ok to include this as a cask and a formula as they seem to do different things. 

We try to avoid this scenario as it can cause confusion for users, who are trying to install one or the other when the token overlaps.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**